src/pageLayouts/frc/Team/GetMember.js

```diff
@@ -82,7 +82,7 @@ function getMentors(teamList) {
       const image =
         member.image &&
         require(`../../../assets/images/frc/members/${subteam}/${
-          member.image
+          member.name.replace(" ", "") + ".png"
```

Comment:
@RobotCrazy this is good, but next time I'd prefer `${member.name.replace(" ", "")}.png` - keep it all in a template literal string